src/main/typescript/components/Footer.tsx

```diff
@@ -25,13 +25,16 @@ const nwo = require("../../resources/img/footer/logo_NWO.png")
 
 const ContactColumn = () => (
     <ul className="contact">
-        <li>Anna van Saksenlaan 51</li>
```

Comment:
why no more address?

Reply:
to be similar to the dans website footer

Reply:
I know, you changed that one as well. So in general: why no more address in both footers.
(unacceptable answer: "_to keep both of them in sync_" 😜 )

Reply:
the dans website is not the webui, but dans.knaw.nl

so, i changed both the deposit-ui and the webui to be more in line with the dans website